### ISSUE TITLE
Save concptarium URL with local cookie

### ISSUE THREAD

**Issue by issmirnov** [paulbricman/shannon-nutri-score#1]
Currently, I have this deployed via docker on a personal server. On every page refresh, I am asked for the url of the conceptarium. It would be useful to have this URL saved in local cookie storage.

**Comment by paulbricman** [comment]
Good point. The Streamlit framework itself resets the session store on page refresh, with apparently [no easy way around it](https://discuss.streamlit.io/t/session-expires-on-page-reload/4651) atm. But it could be saved locally in a text file or something. Will look into it, probably sometime this month.
